doc/prototype/filter-members.md

```diff
@@ -5,46 +5,64 @@
 
```

Comment:
In addition to filtering, it can do pagination and sorting.  We might want to mention that.

Reply:
Added to #16.